pennylane/transforms/batch_params.py

```diff
@@ -71,21 +71,23 @@ def circuit(x, weights):
 
     >>> batch_size = 3
     >>> x = np.linspace(0.1, 0.5, batch_size)
-    >>> weights = np.random.random((batch_size, 10, 3, 3))
+    >>> weights = np.random.random((batch_size, 10, 3, 3), requires_grad=True)
 
     If we evaluate the QNode with these inputs, we will get an output
     of shape ``(batch_size,)``:
 
     >>> circuit(x, weights)
-    [-0.30773348  0.23135516  0.13086565]
+    tensor([[-0.15495184],
+            [-0.30994815],
+            [ 0.34075768]], requires_grad=True)

```

Comment:
@josh146 would this output shape be intended here? :thinking:

Reply:
oh nice catch @antalszava! This should be a 1D array. This can be fixed by changing the line below to

```python
return output_tapes, lambda x: qml.math.squeeze(qml.math.stack(x))
```

Reply:
I've corrected this in dac82b6 :)

Reply:
```suggestion
    tensor([-0.30773348 0.23135516 0.13086565], requires_grad=True)
```